### PR TITLE
updated readme and added action input detail file

### PR DESCRIPTION
**Pull request by Anshul718**
- added all the details of the action in the readme
- added input details in Action_inputs.md
- create-secret part is now optional

### CODE REVIEW

File: Action_inputs.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  <tr>
+    <td><code>manifests</code><br/>Manifests</td>
+    <td>(Required) Path to the manifest files to be used for deployment</td>
+  </tr>
+  <tr>
+    <td><code>images</code><br/>Images</td>
+    <td>(Optional) Fully qualified resource URL of the image(s) to be used for substitutions on the manifest files. This multiline input accepts specifying multiple artifact substitutions in newline separated form. For example - <br>images: |<br>&nbsp&nbspcontosodemo.azurecr.io/foo:test1<br>&nbsp&nbspcontosodemo.azurecr.io/bar:test2<br>In this example, all references to contosodemo.azurecr.io/foo and contosodemo.azurecr.io/bar are searched for in the image field of the input manifest files. For the matches found, the tags test1 and test2 are substituted.</td>
+  </tr>
+  <tr>
+    <td><code>imagepullsecrets</code><br/>Image pull secrets</td>
```
Comment (pavanvamsi3) [inline Action_inputs.md:92]: Remove the Extra description of the code if not required. Put the extra description in brackets.

File: Action_inputs.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
+## Action inputs
+
+### Inputs specific to set cluster context
+<table>
+  <thead>
+    <tr>
+      <th>Action inputs</th>
+      <th>Description</th>
```
Comment (rgsubh) [inline Action_inputs.md:8]: Please have a septate column for 'required'/'optional'

File: Action_inputs.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  <tr>
+    <td><code>kubeconfig</code><br/>Kubectl config</td>
+    <td>(Relevant for kubeconfig approach) Contents of the configuration file to be used with kubectl (e.g. can be pulled from a secret)</td>
+  </tr>
+  <tr>
+    <td><code>context</code><br/>Context</td>
+    <td>(Relevant for kubeconfig approach) Context to be used within the provided kubeconfig file</td>
+  </tr>
+  <tr>
+    <td><code>k8s-url</code><br/>API server URL</td>
```
Comment (rgsubh) [inline Action_inputs.md:24]: you can have the 'API server URL' in a sperate column and name it appropriately

File: Action_inputs.md (hunk 1 above)
Comment (Anshul718) [reply Action_inputs.md:110]: I tried to minimize the description as much as possible for all inputs. Please review the changes.

File: Action_inputs.md (hunk 2 above)
Comment (Anshul718) [reply Action_inputs.md:11]: done

File: Action_inputs.md (hunk 3 above)
Comment (Anshul718) [reply Action_inputs.md]: I replaced it with the 'API server address' as this term is used on Azure Kubernetes Services portal.

**Comment by ds-ms** [comment]
Removing myself from the reviews.. I was more interested in the implementation side of things.. I'll take a look at your other PRs. 😄

**Pull request merged**